### PR TITLE
Pushes for new subscriptions

### PR DESCRIPTION
**Pull request by or-else**
Also AndroidNotification refactored. This refactoring is not directly related to the new push type.

### CODE REVIEW

File: server/push/fcm/push_fcm.go [hunk 1]
```diff
@@ -37,15 +37,112 @@ type Handler struct {
 	client *fcm.Client
 }
 
+// Configuration of AndroidNotification payload.
+type androidConfig struct {
```
Comment (aforge) [inline server/push/fcm/push_fcm.go]: Can you move it closer to other struct definitions for better readability please?

File: server/push/fcm/push_fcm.go [hunk 2]
```diff
@@ -175,6 +236,19 @@ func sendNotifications(rcpt *push.Receipt, config *configType) {
...
 	}
 
+	var titlelc, title, bodylc, body, icon, color string
+	if config.Android.Enabled {
+		titlelc = config.Android.getTitleLocKey(rcpt.Payload.What)
+		title = config.Android.getTitle(rcpt.Payload.What)
+		bodylc = config.Android.getBodyLocKey(rcpt.Payload.What)
+		body = config.Android.getBody(rcpt.Payload.What)
+		if body == "$content" {
+			body = data["content"]
```
Comment (aforge) [inline server/push/fcm/push_fcm.go]: Should we also check if `body` is empty here?

Comment (or-else) [reply server/push/fcm/push_fcm.go]: Maybe it's intentionally blank.

File: server/push/fcm/push_fcm.go (hunk 1 above)
Comment (or-else) [reply server/push/fcm/push_fcm.go]: The logic I followed was type definition immediately followed by all the methods of the type, then the next type definition. I can do [type def 1], [type def 2]..., [methods for type 1], etc but I don't think it's objectively better, just a matter of preference.

**Pull request merged**